### PR TITLE
Release 1.6.0

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by popomore** [comment]
改下版本号？

**Comment by fengmk2** [comment]
@popomore 改好了。

File: History.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+## 2017-07-04, Version 1.6.0, @fengmk2
+
+### Notable changes
+
+* **feature**
+  * tsd add ctx.logger and logger.error support Error object
+* **fix**
+  * don't cache the intermediate locals for application
+  * ignore any key contains "secret" on dump config files
+  * show who define the property of the config on `run/application_config_meta.json`
```
Comment (popomore) [inline History.md]: 这个算 feature 吧

File: History.md [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
 
+## 2017-07-04, Version 1.6.0, @fengmk2
+
+### Notable changes
+
+* **feature**
+  * tsd add ctx.logger and logger.error support Error object
+* **fix**
+  * don't cache the intermediate locals for application
+  * ignore any key contains "secret" on dump config files
```
Comment (popomore) [inline History.md]: 这个算 improve

Comment (fengmk2) [inline History.md]: 嗯，放错地方了。

**Pull request merged**